src/ClientModel/CaptureData.cpp

```diff
@@ -111,6 +111,37 @@ void CaptureData::UpdateFunctionStats(uint64_t instrumented_function_id, uint64_
   }
 }
 
+void CaptureData::OnCaptureComplete(std::vector<orbit_client_data::TimerChain*> chains) {
+  // Recalculate standard deviation as the running calculation may have introduced error.
+  for (auto& pair : functions_stats_) {
+    FunctionStats& stats = pair.second;
+    stats.set_variance_ns(0);
+  }
+
+  for (auto& chain : chains) {
+    if (!chain) continue;
```

Comment:
We should CHECK here. It used to be the case that we could have nullptr chains, but it shouldn't be the case anymore. Dimitry recently added CHECKs in a lot of places where we are using this pattern.

Reply:
done